integration_tests/commands/websocket/setup.go

```diff
@@ -0,0 +1,128 @@
+package websocket
+
+import (
+	"context"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"log"
+	"log/slog"
+	"net/http"
+	"sync"
+	"time"
+
+	"github.com/dicedb/dice/config"
+	"github.com/dicedb/dice/internal/server"
+	"github.com/dicedb/dice/internal/shard"
+	dstore "github.com/dicedb/dice/internal/store"
+	"github.com/gorilla/websocket"
+)
+
+const url = "ws://localhost:8380"
+
+type TestServerOptions struct {
+	Port   int
+	Logger *slog.Logger
+}
+
+type CommandExecutor interface {
+	FireCommand(cmd string) interface{}
+	Name() string
+}
+
+type WebsocketCommandExecutor struct {
```

Comment:
Thank you for using the CommandExecutor! The idea was to have a standard interface to handle tests.
This way, we can combine the actual tests later.